lib/components/transaction_form.dart

```diff
@@ -15,33 +15,30 @@ class _TransactionFormState extends State<TransactionForm> {
 
   final valueController = TextEditingController();
 
- DateTime _selectedDate = DateTime.now();
+  DateTime _selectedDate = DateTime.now();
 
-  _submitForm() {
+  void _submitForm() {
```

Comment:
Sempre adicione valores de retorno para métodos de classe, se não for retornar nada, use `void`, essa é uma styleguide recomendada e utilizada para Dart. Caso tenha interesse e ainda não tenha visto, o `flutter_lints` é um pacote Dart que enumera e automatiza todos esses estilos no código. Esse pacote [está disponível aqui.](https://pub.dev/packages/flutter_lints)